src/raftstore/store/worker/region.rs

```diff
@@ -48,6 +50,10 @@ const GENERATE_POOL_SIZE: usize = 2;
 
 // used to periodically check whether we should delete a stale peer's range in region runner
 pub const STALE_PEER_CHECK_INTERVAL: u64 = 10_000; // milliseconds
+
+// used to periodically check whether schedule pending applies in region runner
+pub const PENDING_APPLY_CHECK_INTERVAL: u64 = 1_000; // milliseconds
```

Comment:
1ms or 1000ms? Does it configurable?

Reply:
I check other similar intervals, they use const too. And I don't think it should be configurable.